rocky/components/modal/style.scss

```diff
@@ -0,0 +1,82 @@
+dialog {
+  border: 1px solid var(--colors-grey-4);
+  border-radius: 8px;
+  max-height: calc(100vh - 3rem);
+  padding: 0;
+
+  &.dialog-small {
+    width: 27.5rem;
+
+    .content-wrapper {
+      .header,
+      .footer {
+        border: 0;
+      }
+
+      .content {
+        padding-top: 0;
+        padding-bottom: 0;
+      }
+    }
+  }
+
+  &.dialog-medium {
+    width: 50rem;
+  }
+
+  &.dialog-large {
+    width: 71.25rem;
+  }
+
+  &::backdrop {
+    background-color: rgb(18 21 23 / 50%);
```

Comment:
Can this be a variable color from the theme?

Reply:
Will look into this with someone from UX.